src/SprykerEco/Yves/Payone/PayoneFactory.php

```diff
@@ -388,4 +416,12 @@ public function createQuoteHydrator()
             $this->getCalculationClient()
         );
     }
+
+    /**
+     * @return \Generated\Shared\Transfer\PayoneKlarnaStartSessionRequestTransfer
+     */
+    public function createPayoneKlarnaStartSessionRequest(): PayoneKlarnaStartSessionRequestTransfer
```

Comment:
Transfers are allowed to be instantiated outside the factory. An unnecessary method

Reply:
Removed